build.gradle

```diff
@@ -113,6 +113,7 @@ allprojects {
     configurations.all {
         resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib:1.6.0"
         resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib-common:1.6.0"
+        resolutionStrategy.force 'org.apache.commons:commons-lang3:3.13.0'
```

Comment:
@vamsi-amazon I believe you need to exclude the `commons-lang3` since it is part of core :(

Reply:
@reta thanks will look into it.

Reply:
@reta  can i find all the dependencies in core at one place? 
should i go through all the gradle files or build and look into libs folder?